etc/docker/README.md

```diff
@@ -11,57 +11,23 @@ It provides the following containers:
 
 ## Installation
 
-1. Create a `.env` from the `.env.dist` file. Adapt it according to your symfony application
-
-    ```bash
-    cp .env.dist .env
-    ```
+1. Adapt `.env.dist` file according to your symfony application. It will be copied to `.env` by upstart script.
 
 
-2. Build/run containers with (with and without detached mode)
+2. Update your system host file
 
+    Add this line to /etc/hosts 
     ```bash
-    $ docker-compose build
-    $ docker-compose up -d
+    "127.0.0.1 kreta.localhost taskmanager.localhost identityaccess.localhost"
```

Comment:
And in troubleshooting section add 502 unexpected behaviour explanation until we get a better choice for run node